src/pybind/mgr/orchestrator/module.py

```diff
@@ -1411,12 +1424,19 @@ def _daemon_add_osd(self,
     def _osd_rm_start(self,
                       osd_id: List[str],
                       replace: bool = False,
+                      replace_block: bool = False,
+                      replace_db: bool = False,
+                      replace_wal: bool = False,
```

Comment:
for my clarification, is this meant to be the mechanism to replace a device by osd id, while the `orch device replace` is for doing it by path? We may also want to note these new fields in the docs change.

Reply:
legacy from previous approach